lib/cartopy/tests/mpl/test_mpl_integration.py

```diff
@@ -132,6 +132,36 @@ def test_global_scatter_wrap_no_transform():
     plt.scatter(x, y, c=data)
 
 
+@pytest.mark.natural_earth
+@ImageTesting(['global_hexbin_wrap'],
+              tolerance=12.4 if MPL_VERSION < '2.1.0' else 0.5)
+def test_global_hexbin_wrap_new_transform():
+    ax = plt.axes(projection=ccrs.PlateCarree())
+    # By default the coastline feature will be drawn after patches.
+    # By setting zorder we can ensure our hexbin polygons are drawn
+    # after the coastlines.
+    ax.coastlines(zorder=0)
+    x, y = np.meshgrid(np.linspace(0, 360), np.linspace(-90, 90))
+    data = np.sin(np.sqrt(x ** 2 + y ** 2))
```

Comment:
How would you feel about changing the image data to: `sqrt((x - 180)**2 + y**2)`? This makes the plot look nice at least I think because then the wrapped values don't show a discontinuity due to bin edges being weird.

Reply:
I ended up going to x values between -180 and 180, so I think remediates your suggestion.